libraries/AP_HAL_ChibiOS/hwdef/F4BY/hwdef.dat

```diff
@@ -162,3 +162,5 @@ PC9  EXTERN_GPIO5 OUTPUT GPIO(5)
 #  IRQ for MPU6000
 PB0 EXTI_MPU6000 INPUT PULLUP
 PB1 DRDY_HMC5883 INPUT PULLUP
+
+AUTOBUILD_TARGETS Copter
```

Comment:
This doesn't make sense to me; this is a normal-looking board which could be used in a Plane.  I've got one.

Reply:
its only was ever advertised as having copter ardupilot firmware and is been discontinued and not available for a while

Reply:
changed to minimize_features and all builds